packages/trader/src/AppV2/Utils/sort-symbols-utils.ts

```diff
@@ -0,0 +1,27 @@
+import { ActiveSymbols } from '@deriv/api-types';
+
+type MarketOrderMap = {
+    [key: string]: number;
+};
+
+const sortSymbols = (symbolsList: ActiveSymbols) => {
+    const marketSortingOrder = ['synthetic_index', 'forex', 'indices', 'cryptocurrency', 'commodities'];
+    const marketOrderMap: MarketOrderMap = marketSortingOrder.reduce(
+        (acc: MarketOrderMap, market: string, index: number) => {
+            acc[market] = index;
+            return acc;
+        },
+        {}
+    );
+
+    return symbolsList.sort((a, b) => {
+        const marketOrderA = marketOrderMap[a.market] !== undefined ? marketOrderMap[a.market] : symbolsList.length;
```

Comment:
`const marketOrderA = marketOrderMap[a.market] || (or ?? , depends on what is there) symbolsList.length;`

Reply:
marketOrderMap[a.market] is a number/index based on ordering that i provided.  if the thing 'market' doesnt exist in the ordering we put it in the very last spot hence symbolsList.length